packages/shared/modals/Selector.tsx

```diff
@@ -170,11 +170,14 @@ export class SelectorInput extends React.Component<
     };
   }
 
-  static getDerivedStateFromProps(props, state) {
-    if (!_.isEqual(props.tags, state.tags)) {
-      return { ...state, tags: props.tags };
+  componentDidUpdate(prevProps) {
+    if (!_.isEqual(prevProps.tags, this.props.tags)) {
+      this.updateProp(this.props.tags);
     }
-    return state;
+  }
+
+  updateProp(tags: string[]) {
```

Comment:
Shouldn't this be called `updateTags` ?
```suggestion
  priavte updateTags(tags: string[]) {
```